### ISSUE TITLE
Wilder popup menu does not close after pressing <C-f>

### ISSUE THREAD

**Issue by Melkster** [gelguy/wilder.nvim#70]
By default `<C-f>` opens the command-line window. However, wilder leaves its popup menu open when pressing `<C-f>`, which blocks the command-line-window.

**To reproduce:**

1. Use the following minimal `.vimrc`:

```vim
call plug#begin('~/.config/nvim/packages/')
Plug 'gelguy/wilder.nvim'
call plug#end()

call wilder#setup({'modes': [':', '/', '?']})
call wilder#set_option('renderer', wilder#popupmenu_renderer({
      \ 'highlighter': wilder#basic_highlighter(),
      \ }))
```

2. Do `:` and then press `<C-f>`.

**Comment by gelguy** [comment]
Thanks for reporting! I can reproduce this.

I've just pushed a fix for this in the latest master: bc46e7101c179bf1f2588c13663be028495a53b1. Let me know if this fixes the bug.

**Comment by Melkster** [comment]
Yep that seems to do the trick. Thanks! :)
